roles/config-docker-compose/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
This content isn't used anywhere in this role. Did you leave out some changes? 
Also, did you mean for this to be `docker_compose_install`? If so, it should default to `false`